pkg/theia/commands/anomaly_detection_run.go

```diff
@@ -240,5 +296,34 @@ Example values include 0.1, 500m, 1.5, 5, etc.`,
 		`Specify the memory request for the executor Pod. Values conform to the Kubernetes resource quantity convention.
 Example values include 512M, 1G, 8G, etc.`,
 	)
-
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"agg-flow",
+		"",
+		`Specifies which aggregated flow to perform anomaly detection on, options are pod/svc/external`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"pod-label",
+		"",
+		`On choosing agg-flow as pod, user has option to mention labels for inbound/outbound throughput, default would be all labels`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"pod-name",
+		"",
+		`On choosing agg-flow as pod, user has option to mention podname for inbound/outbound throughput, if pod-labels are mentioned, that will take priority`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"pod-namespace",
+		"",
+		`On choosing agg-flow as pod, user has option to mention podnamespace for inbound/outbound throughput, podnamespace argument should be combined with podlabels or podname`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"external-ip",
+		"",
+		`On choosing agg-flow as external, user has option to mention external-ip for inbound throughput, default would be all IPs`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"svc-port-name",
+		"",
+		`On choosing agg-flow as svc, user has option to mention svc-port-name for inbound throughput, default would be all service port names`,
+	)
```

Comment:
I might be wrong, but would like to double check with you is this inbound or outbound? As I see the externalIp and svc port name are used to filter the destination, should that be outbound?

Reply:
Yes this is outbound, Thanks for noticing